lib/controller/matlab/wb_altimeter_disable.m

```diff
@@ -0,0 +1,6 @@
+function wb_altimeter_disable(tag)
+% Usage: wb_altimeter_disable(tag)
+% Matlab API for Webots
+% Online documentation is available <a href="https://www.cyberbotics.com/doc/reference/altimeter">here</a>
+
+calllib('libController', 'wb_altimeter_disable', tag);
```

Comment:
Given a very recent change, it is no longer necessary to push the `.m` files.
Same thing for the other 3.